include/types/register.hpp

```diff
@@ -49,4 +49,11 @@ using BaseHalfTileColMajor = RegTile<cutlass::half_t, tl::ColMajor<4, 2>>;
 
 using BaseFloatTileRowMajor = RegTile<float, tl::RowMajor<2, 4>>;
 using BaseFloatTileColMajor = RegTile<float, tl::ColMajor<4, 2>>;
+
+template <typename Element>
+using BaseTileRowMajor = RegTile<Element, tl::RowMajor<2, 4>>;
+
+template <typename Element>
+using BaseTileColMajor = RegTile<Element, tl::ColMajor<4, 2>>;
```

Comment:
Should we delete lines 47-51 and keep lines 53-57 instead, constraining the type to `cutlass::half_t`, `float`, or others?

It seems that lines 53-56 generalize the type definition, making the specialization no longer useful.

Reply:
Removed the redundant definition of `BaseTile`.